netbox/netbox_inventory.py

```diff
@@ -71,57 +80,98 @@ def cli():
 
 
 class NetBoxSession(requests.Session):
-    def __init__(self, url, token):
+    """A session for interacting with the NetBox API.
+
+    Attributes:
+        url: The base URL of the NetBox instance.
+        token: The API token for authentication.
+    """
+
+    def __init__(self, url: str, token: str):
+        """Initialize the NetBoxSession.
+
+        Args:
+            url: The base URL of the NetBox instance.
+            token: The API token for authentication.
+        """
         super(NetBoxSession, self).__init__()
         self.url = url
-        self.headers["authorization"] = "Token %s" % token
+        self.headers["authorization"] = f"Token {token}"
         self.verify = False
```

Comment:
To enhance security, consider enabling SSL verification or providing a way to configure it through environment variables. [important]